src/Uno.UI/UI/Xaml/Controls/ToolTip/ToolTip.cs

```diff
@@ -40,6 +40,7 @@ internal Popup Popup
 					_popup.PopupPanel = new PopupPanel(_popup);
 					_popup.Opened += OnPopupOpened;
 					_popup.Closed += (sender, e) => IsOpen = false;
+					_popup.DataContext = DataContext;
```

Comment:
Did you try adding what I mentioned here: https://github.com/unoplatform/uno/issues/9570#issuecomment-1227190049 ?

Reply:
@jeromelaban Yeah, OnDataContextChanged isn't called.

The reason is when it gets changed, the precedence is `Inheritance`, but it already has a `Local` precedence.